webextensions/api/bookmarks.json

```diff
@@ -94,7 +94,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/CreateDetails",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "36"
```

Comment:
✅ https://github.com/chromium/chromium/commit/eb59ad1f79b303f1091ea5b13075825fc81b8ece in [36.0.1955.0](https://github.com/chromium/chromium/releases/tag/36.0.1955.0). before 36.0.1985